### ISSUE TITLE
Exception during phpstorm inspection

### ISSUE THREAD

**Issue by asgraf** [dmeybohm/chocolate-cakephp#174]
I got this error everytime i run inspections on my whole project
```
java.lang.StringIndexOutOfBoundsException: Range [3, 2) out of bounds for length 2
	at java.base/jdk.internal.util.Preconditions$1.apply(Preconditions.java:55)
	at java.base/jdk.internal.util.Preconditions$1.apply(Preconditions.java:52)
	at java.base/jdk.internal.util.Preconditions$4.apply(Preconditions.java:213)
	at java.base/jdk.internal.util.Preconditions$4.apply(Preconditions.java:210)
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:98)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckFromToIndex(Preconditions.java:112)
	at java.base/jdk.internal.util.Preconditions.checkFromToIndex(Preconditions.java:349)
	at java.base/java.lang.String.checkBoundsBeginEnd(String.java:4865)
	at java.base/java.lang.String.substring(String.java:2834)
	at java.base/java.lang.String.substring(String.java:2807)
	at com.daveme.chocolateCakePHP.model.AssociatedTableTypeProvider.complete(AssociatedTableTypeProvider.kt:56)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:252)
	at com.jetbrains.php.PhpIndexImpl.lambda$doCompleteType$1(PhpIndexImpl.java:181)
	at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:111)
	at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:27)
	at com.jetbrains.php.PhpIndexImpl.doCompleteType(PhpIndexImpl.java:181)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:131)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.decodeExtendedClassesAndSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:126)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.decodeExtendedClassesAndSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:96)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.getExtendedClassesToSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:88)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.complete(PhpGenericsBaseExtendedWithGenericTypeProvider.java:48)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:251)
	at com.jetbrains.php.PhpIndexImpl.lambda$doCompleteType$1(PhpIndexImpl.java:181)
	at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:111)
	at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:27)
	at com.jetbrains.php.PhpIndexImpl.doCompleteType(PhpIndexImpl.java:181)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:131)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.decodeExtendedClassesAndSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:126)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.decodeExtendedClassesAndSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:96)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.getExtendedClassesToSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:88)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.complete(PhpGenericsBaseExtendedWithGenericTypeProvider.java:48)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:251)
	at com.jetbrains.php.PhpIndexImpl.lambda$doCompleteType$1(PhpIndexImpl.java:181)
	at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:111)
	at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:27)
	at com.jetbrains.php.PhpIndexImpl.doCompleteType(PhpIndexImpl.java:181)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:131)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.decodeExtendedClassesAndSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:126)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.decodeExtendedClassesAndSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:96)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.getExtendedClassesToSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:88)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.complete(PhpGenericsBaseExtendedWithGenericTypeProvider.java:48)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:251)
	at com.jetbrains.php.PhpIndexImpl.lambda$doCompleteType$1(PhpIndexImpl.java:181)
	at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:111)
	at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:27)
	at com.jetbrains.php.PhpIndexImpl.doCompleteType(PhpIndexImpl.java:181)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:131)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.decodeExtendedClassesAndSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:126)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.decodeExtendedClassesAndSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:96)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.getExtendedClassesToSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:88)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.complete(PhpGenericsBaseExtendedWithGenericTypeProvider.java:48)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:251)
	at com.jetbrains.php.PhpIndexImpl.lambda$doCompleteType$1(PhpIndexImpl.java:181)
	at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:111)
	at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:27)
	at com.jetbrains.php.PhpIndexImpl.doCompleteType(PhpIndexImpl.java:181)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:131)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.decodeExtendedClassesAndSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:126)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.decodeExtendedClassesAndSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:96)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.getExtendedClassesToSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:88)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.complete(PhpGenericsBaseExtendedWithGenericTypeProvider.java:48)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:251)
	at com.jetbrains.php.PhpIndexImpl.lambda$doCompleteType$1(PhpIndexImpl.java:181)
	at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:111)
	at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:27)
	at com.jetbrains.php.PhpIndexImpl.doCompleteType(PhpIndexImpl.java:181)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:131)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.decodeExtendedClassesAndSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:126)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.decodeExtendedClassesAndSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:96)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.getExtendedClassesToSubstitutedTemplates(PhpGenericsBaseExtendedWithGenericTypeProvider.java:88)
	at com.jetbrains.php.lang.psi.resolve.types.generics.PhpGenericsBaseExtendedWithGenericTypeProvider.complete(PhpGenericsBaseExtendedWithGenericTypeProvider.java:48)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:251)
	at com.jetbrains.php.PhpIndexImpl.lambda$doCompleteType$1(PhpIndexImpl.java:181)
	at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:111)
	at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:27)
	at com.jetbrains.php.PhpIndexImpl.doCompleteType(PhpIndexImpl.java:181)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:131)
	at com.jetbrains.php.PhpIndexImpl.getClasses(PhpIndexImpl.java:332)
	at com.jetbrains.php.PhpIndexImpl.tryAddMockeryType(PhpIndexImpl.java:266)
	at com.jetbrains.php.PhpIndexImpl.doCompleteType(PhpIndexImpl.java:207)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:131)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:104)
	at com.jetbrains.php.lang.psi.resolve.types.PhpType.global(PhpType.java:1077)
	at com.jetbrains.php.lang.psi.resolve.types.PhpExcludeTypeTP.complete(PhpExcludeTypeTP.java:54)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:252)
	at com.jetbrains.php.PhpIndexImpl.lambda$doCompleteType$1(PhpIndexImpl.java:181)
	at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:111)
	at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:27)
	at com.jetbrains.php.PhpIndexImpl.doCompleteType(PhpIndexImpl.java:181)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:131)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:104)
	at com.jetbrains.php.lang.psi.resolve.types.PhpType.global(PhpType.java:1077)
	at com.jetbrains.php.lang.psi.elements.PhpTypedElement.getGlobalType(PhpTypedElement.java:57)
	at com.jetbrains.php.lang.psi.elements.impl.MemberReferenceImpl.resolveInHierarchyInner(MemberReferenceImpl.java:194)
	at com.jetbrains.php.lang.psi.elements.impl.MemberReferenceImpl.lambda$resolveInHierarchy$1(MemberReferenceImpl.java:187)
	at com.intellij.psi.util.CachedValuesManager$1.compute(CachedValuesManager.java:173)
	at com.intellij.psi.impl.PsiCachedValueImpl$Direct.doCompute(PsiCachedValueImpl.kt:77)
	at com.intellij.util.CachedValueBase.lambda$getValueWithLock$3(CachedValueBase.java:236)
	at com.intellij.util.CachedValueBase.computeData(CachedValueBase.java:43)
	at com.intellij.util.CachedValueBase.lambda$getValueWithLock$4(CachedValueBase.java:236)
	at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:111)
	at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:27)
	at com.intellij.openapi.util.RecursionManager.doPreventingRecursion(RecursionManager.java:66)
	at com.intellij.util.CachedValueBase.getValueWithLock(CachedValueBase.java:237)
	at com.intellij.psi.impl.PsiCachedValueImpl$Direct.getValue(PsiCachedValueImpl.kt:81)
	at com.intellij.util.CachedValuesManagerImpl.getCachedValue(CachedValuesManagerImpl.java:83)
	at com.intellij.psi.util.CachedValuesManager.getCachedValue(CachedValuesManager.java:170)
	at com.intellij.psi.util.CachedValuesManager.getCachedValue(CachedValuesManager.java:136)
	at com.jetbrains.php.lang.psi.elements.impl.MemberReferenceImpl.resolveInHierarchy(MemberReferenceImpl.java:186)
	at com.jetbrains.php.lang.psi.elements.impl.MemberReferenceImpl.resolveInHierarchy(MemberReferenceImpl.java:178)
	at com.jetbrains.php.lang.inspections.PhpPossiblePolymorphicInvocationInspection$1.check(PhpPossiblePolymorphicInvocationInspection.java:74)
	at com.jetbrains.php.lang.inspections.PhpPossiblePolymorphicInvocationInspection$1.visitPhpFieldReference(PhpPossiblePolymorphicInvocationInspection.java:68)
	at com.jetbrains.php.lang.psi.elements.impl.FieldReferenceImpl.accept(FieldReferenceImpl.java:201)
	at com.jetbrains.php.lang.psi.elements.impl.PhpASTElementImpl.accept(PhpASTElementImpl.java:26)
	at com.intellij.codeInsight.daemon.impl.InspectionVisitorOptimizer.lambda$acceptElements$1(InspectionVisitorOptimizer.java:187)
	at com.intellij.codeInsight.daemon.impl.InspectionVisitorOptimizer.acceptElements(InspectionVisitorOptimizer.java:207)
	at com.intellij.codeInsight.daemon.impl.InspectionVisitorOptimizer.acceptElements(InspectionVisitorOptimizer.java:187)
	at com.intellij.codeInspection.InspectionEngine.lambda$inspectElements$8(InspectionEngine.java:284)
	at com.intellij.concurrency.ApplierCompleter.processArrayItem(ApplierCompleter.java:116)
	at com.intellij.concurrency.ApplierCompleter.processArray(ApplierCompleter.java:196)
	at com.intellij.concurrency.ApplierCompleter.execAll(ApplierCompleter.java:166)
	at com.intellij.concurrency.JobLauncherImpl.lambda$invokeConcurrentlyUnderProgressAsync$3(JobLauncherImpl.java:103)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.tryRunReadAction(AnyThreadWriteThreadingSupport.kt:279)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:965)
	at com.intellij.concurrency.ApplierCompleter.lambda$wrapInReadActionAndIndicator$2(ApplierCompleter.java:148)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:660)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:735)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:691)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:659)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:79)
	at com.intellij.concurrency.ApplierCompleter.wrapInReadActionAndIndicator(ApplierCompleter.java:159)
	at com.intellij.concurrency.ApplierCompleter.wrapAndRun(ApplierCompleter.java:143)
	at com.intellij.concurrency.JobLauncherImpl.lambda$invokeConcurrentlyUnderProgressAsync$4(JobLauncherImpl.java:103)
	at com.intellij.concurrency.JobLauncherImpl.safeIterate(JobLauncherImpl.java:168)
	at com.intellij.concurrency.JobLauncherImpl.invokeConcurrentlyUnderProgressAsync(JobLauncherImpl.java:100)
	at com.intellij.concurrency.JobLauncherImpl.invokeConcurrentlyUnderProgress(JobLauncherImpl.java:48)
	at com.intellij.concurrency.JobLauncher.invokeConcurrentlyUnderProgress(JobLauncher.java:51)
	at com.intellij.codeInspection.InspectionEngine.lambda$inspectElements$9(InspectionEngine.java:324)
	at com.intellij.codeInspection.InspectionEngine.withSession(InspectionEngine.java:205)
	at com.intellij.codeInspection.InspectionEngine.inspectElements(InspectionEngine.java:245)
	at com.intellij.codeInspection.InspectionEngine.inspectElements(InspectionEngine.java:193)
	at com.intellij.codeInspection.InspectionEngine.inspectEx(InspectionEngine.java:99)
	at com.intellij.codeInspection.ex.GlobalInspectionContextImpl.lambda$inspectFile$15(GlobalInspectionContextImpl.java:542)
	at com.intellij.codeInspection.ex.InspectionProfileWrapper.runWithCustomInspectionWrapper(InspectionProfileWrapper.java:90)
	at com.intellij.codeInspection.ex.GlobalInspectionContextImpl.lambda$inspectFile$16(GlobalInspectionContextImpl.java:539)
	at com.intellij.codeInsight.daemon.impl.HighlightingSessionImpl.runInsideHighlightingSession(HighlightingSessionImpl.java:179)
	at com.intellij.codeInspection.ex.GlobalInspectionContextImpl.inspectFile(GlobalInspectionContextImpl.java:538)
	at com.intellij.codeInspection.ex.GlobalInspectionContextImpl.lambda$buildProcessor$5(GlobalInspectionContextImpl.java:418)
	at com.intellij.openapi.project.DumbService.runReadActionInSmartMode$lambda$0(DumbService.kt:83)
	at com.intellij.openapi.project.DumbService.runReadActionInSmartMode$lambda$2(DumbService.kt:135)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runReadAction(AnyThreadWriteThreadingSupport.kt:264)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runReadAction(AnyThreadWriteThreadingSupport.kt:221)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:857)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:66)
	at com.intellij.openapi.project.DumbService.runReadActionInSmartMode(DumbService.kt:128)
	at com.intellij.openapi.project.DumbService.runReadActionInSmartMode(DumbService.kt:83)
	at com.intellij.openapi.project.DumbService.tryRunReadActionInSmartMode(DumbService.kt:109)
	at com.intellij.codeInspection.ex.GlobalInspectionContextImpl.lambda$buildProcessor$9(GlobalInspectionContextImpl.java:428)
	at com.intellij.concurrency.JobLauncherImpl$1MyProcessQueueTask.lambda$call$0(JobLauncherImpl.java:382)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:660)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:735)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:691)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:659)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:79)
	at com.intellij.concurrency.JobLauncherImpl$1MyProcessQueueTask.call(JobLauncherImpl.java:367)
	at com.intellij.concurrency.JobLauncherImpl$1MyProcessQueueTask.call(JobLauncherImpl.java:353)
	at java.base/java.util.concurrent.ForkJoinTask$AdaptedInterruptibleCallable.compute(ForkJoinTask.java:1689)
	at java.base/java.util.concurrent.ForkJoinTask$InterruptibleTask.exec(ForkJoinTask.java:1641)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:507)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1491)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:2073)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:2035)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:187)
```
PhpStorm version:
```
PhpStorm 2024.2.3
Build #PS-242.23339.16, built on September 25, 2024
Licensed to Azagroup S.A. / Michał Turek
Subscription is active until May 19, 2025.
Runtime version: 21.0.4+13-b509.17 amd64 (JCEF 122.1.9)
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Toolkit: sun.awt.windows.WToolkit
Windows 11.0
GC: G1 Young Generation, G1 Concurrent GC, G1 Old Generation
Memory: 4096M
Cores: 12
Registry:
  run.processes.with.pty=TRUE
  ide.experimental.ui=true
  i18n.locale=
Non-Bundled Plugins:
  cc.duduhuo.git-conflict (2.0.0)
  dev.andrybak.intellij.copy_commit_reference (2.0.0)
  software.xdev.saveactions (1.3.1)
  org.OverEngineer.InlineProblems (0.5.3)
  com.guohanlin.JsonToPhp (2023.1.2)
  String Manipulation (9.14.1)
  Key Promoter X (2024.2.0)
  com.GEN.LastEdited.v1 (1.10.2)
  indent-rainbow.indent-rainbow (2.2.0)
  com.dohyeon5626.auto-gitkeep-plugin (1.5)
  com.jetbrains.changeReminder (242.20224.155)
  com.deepspace.stash_notifications_plugin (1.0.0)
  com.vladsch.git-file-case-fixer (1.0.20)
  izhangzhihao.rainbow.brackets (2024.2.6-241)
  ru.vkurdin.idea.php.lambdafolding (0.3.0)
  com.kalessil.phpStorm.phpInspectionsEA (5.1.0)
  com.github.timo_reymann.composer_dump_autoload_phpstorm_plugin (2.1.1)
  com.funivan.idea.phpClean (2023.12.17)
  org.vk.nocolor (0.0.5)
  com.github.lppedd.idea-return-highlighter (0.8.3)
  com.daveme.intellij.chocolateCakePHP (0.9.1)
  lv.midiana.misc.phpstorm-plugins.deep-keys (2022.08.03.001)
  CakeConfigureCompletion (0.1)
  io.github.linwancen.show-comment (2.16.0.2024.09.26_22.28)
  ru.adelf.idea.dotenv (2024.2.1)
  io.github.freeApplications.intellij-plugins (1.3.0)
  com.funivan.phpstorm.refactoring (0.1.1)
  de.espend.idea.php.toolbox (6.2.0)
  de.espend.idea.php.annotation (11.1.0)
  dev.dohpaz.php-extras (2.2.0)
  net.rentalhost.plugins.php.hammer (0.31.0)
  com.github.copilot (1.5.25.10)
  gr.jchrist.gitextender (0.8.2)
  com.virtuslab.git-machete (5.1.1)
  GitCheckoutFilesFromOtherBranch (0.1.1)
  com.ppolivka.plugin.gitcleaner (1.1.4)
  com.karsta26.recent-projects-with-branch (1.2)
  zielu.gittoolbox (600.0.2+242)
  mobi.hsz.idea.gitignore (4.5.4)
```

**Comment by dmeybohm** [comment]
@asgraf I tried to make a fix for that. Could you try this version by installing it manually and let me know if it works for you? Instructions for installing it are here: https://www.jetbrains.com/help/phpstorm/managing-plugins.html#install_plugin_from_disk

[chocolate-cakephp-0.9.1-test1.zip](https://github.com/user-attachments/files/17608332/chocolate-cakephp-0.9.1-test1.zip)

**Comment by asgraf** [comment]
I can confirm that problem no longer occurs

**Comment by dmeybohm** [comment]
@asgraf Thanks so much for your help. This should be fixed in version 0.9.2, which is on the plugin marketplace now.
